source/testnet/guides/run-node.rst

```diff
@@ -110,14 +109,15 @@ The tool will now download the Concordium Client image and load it into
 Docker. The client will launch and start outputting logging information
 about the operation of the node.
 
+.. _`Netwok Dashboard`: https://dashboard.testnet.concordium.com/
+
+
 Seeing your node on the dashboards
 ==================================
 
 After running ``concordium-node`` you can
 
--  see your node on the """ , networkDashboardLink , """
--  access the node dashboard at `http://localhost:8099`_ (this link
-   becomes active after the node is run)
+-  see your node on the `Network Dashboard`_
```

Comment:
This link appears not to work. It should lead to `https://dashboard.testnet.concordium.com/`